neuralop/datasets/navier_stokes.py

```diff
@@ -1,188 +1,77 @@
-#         x_mean = training_db._attribute('x', 'mean')
-#         x_std = training_db._attribute('x', 'std')
-        
-#         in_normalizer = Normalizer(x_mean, x_std)
-    
-#     if positional_encoding:
-#         pos_encoding = PositionalEmbedding2D(grid_boundaries)
-    
-#     if encode_output:
-#         y_mean = training_db._attribute('y', 'mean')
-#         y_std = training_db._attribute('y', 'std')
+from typing import Union, List
+
+from torch.utils.data import DataLoader
+
+from .pt_dataset import PTDataset
+from .web_utils import download_from_zenodo_record
+
+logger = logging.Logger(logging.root.level)
+
+class DarcyDataset(PTDataset):
+    def __init__(self,
+                 root_dir: Union[Path, str],
+                 n_train: int,
+                 n_tests: List[int],
+                 batch_size: int,
+                 test_batch_sizes: List[int],
+                 train_resolution: int,
+                 test_resolutions: int=[16,32],
+                 grid_boundaries: List[int]=[[0,1],[0,1]],
+                 positional_encoding: bool=True,
+                 encode_input: bool=False, 
+                 encode_output: bool=True, 
+                 encoding="channel-wise",
+                 channel_dim=1,
+                 download: bool=True):
         
-#         out_normalizer = Normalizer(y_mean, y_std)
-
-#     data_processor = DefaultDataProcessor(in_normalizer=in_normalizer,
-#                                           out_normalizer=out_normalizer,
-#                                           positional_encoding=pos_encoding)
-    
-#     train_loader = torch.utils.data.DataLoader(training_db,
-#                                                batch_size=batch_size, 
-#                                                shuffle=True,
-#                                                num_workers=num_workers,
-#                                                pin_memory=pin_memory,
-#                                                persistent_workers=persistent_workers)
-
-#     test_loaders = dict()
-#     for (res, n_test, test_batch_size) in zip(test_resolutions, n_tests, test_batch_sizes):
-#         print(f'Loading test db at resolution {res} with {n_test} samples and batch-size={test_batch_size}')
-
-#         test_db = H5pyDataset(data_path / 'navier_stokes_1024_test.hdf5', n_samples=n_test, resolution=res)
-    
-#         test_loaders[res] = torch.utils.data.DataLoader(test_db, 
-#                                                         batch_size=test_batch_size,
-#                                                         shuffle=False,
-#                                                         num_workers=num_workers, 
-#                                                         pin_memory=pin_memory, 
-#                                                         persistent_workers=persistent_workers)
-
-#     return train_loader, test_loaders, data_processor
-
-
-def load_navier_stokes_pt(data_path, train_resolution,
```

Comment:
Before deleting this function we need to replace its usage in `scripts/`, namely:
https://github.com/neuraloperator/neuraloperator/blob/ae6bdb948b1733a8c1bb862de8127c55c97e3074/scripts/train_navier_stokes.py#L74